bot/controllers/broadcast.js

```diff
@@ -0,0 +1,69 @@
+module.exports = () => {
+  const Discord = require('discord.js');
+  const messageMiddleware = (message) => {
+    const container = {};
+    container.parsed = message.content.split(' ');
+    const msg = Object.assign(message, container);
+    return msg;
+  };
+  // send message for all channels
+  const _allChannels = (message) => {
+    const msg = messageMiddleware(message);
+    const input = msg.parsed.join(',').replace(/,/g, ' ');
+    if (msg.parsed[0].toLowerCase() === '!broadcastall') {
+      const channels = message.guild.channels.map(channel => channel);
+      Object.keys(channels).forEach((el) => {
+        // filter for text channels
+        if (channels[el].type === 'text') {
+          // creating an object to send a message
+          const objEmbed = new Discord.RichEmbed()
```

Comment:
Semantically objEmbed is, again, generic. Try to find a name that fits the context of this situation of its use.

Reply:
would suggest something like `botMsg` or something that lets us know what it is and who/what is sending it.